### DISCUSSION TITLE
Compiling with Clang does not find blitz/llvm/bzconfig.h

### DISCUSSION THREAD

**Discussion by remod** [blitzpp/blitz#178] [General]
When compiling my library which includes Blitz++ with Clang, I am getting the following error:

```
In file included from /usr/include/blitz/array.h:37:
In file included from /usr/include/blitz/array-impl.h:46:
In file included from /usr/include/blitz/blitz.h:52:
/usr/include/blitz/bzconfig.h:82:10: fatal error: 'blitz/llvm/bzconfig.h' file not found
#include <blitz/llvm/bzconfig.h>
         ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

The file is indeed missing in my system, I only seem to have support for gcc (`blitz/gnu/bzconfig.h` exists & works).

Installed version of Blitz++:

```
libblitz0-dev:
  Installed: 1:1.0.2+ds-2
  Candidate: 1:1.0.2+ds-2
  Version table:
 *** 1:1.0.2+ds-2 500
        500 http://ch.archive.ubuntu.com/ubuntu focal/universe amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by slayoo** [comment]
Thanks for the report. Indeed, the Debian/Ubuntu package seem to only ship GCC config file (https://packages.debian.org/sid/amd64/libblitz0-dev/filelist). Would installing Blitz++ from the source tarball using Clang during setup help?

**Comment by remod** [comment]
Yes technically that would probably work!

I am writing a plugin library which supports GCC and Clang compilers and depends on Blitz++. I am providing my library as a Debian package as well, and of course it would be nice if it could just depend on `libblitz0-dev` such that `apt` installs it automatically. I guess for now I'll just leave away the Clang support for my plugin library.

Thanks for your answer!
